src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_message_is_moved_to_error_queue_using_dtc.cs

```diff
@@ -0,0 +1,102 @@
+﻿namespace NServiceBus.AcceptanceTests.Recoverability.Retries
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using System.Transactions;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+    using ScenarioDescriptors;
+
+    public class When_message_is_moved_to_error_queue_using_dtc : NServiceBusAcceptanceTest
```

Comment:
@timbussmann I agree that it should behave in the same way. What would be the value of removing this test? IMO it gives us additional safety net.